lib/factory_bot/null_object.rb

```diff
@@ -6,11 +6,8 @@ def initialize(methods_to_respond_to)
     end
 
     def method_missing(name, *args, &block)
-      if respond_to?(name)
```

Comment:
This seems unrelated to this PR. Also, I think I like the first version a little better. It is easier to see the conditional.